<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaccaratUI</class>
 <widget class="QWidget" name="BaccaratUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 255, 116);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ff0000;&quot;&gt;庄&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;闲&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_Logo">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../imgage.qrc">:/img/baccarat/baijiale.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>350</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>600</width>
       <height>350</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 217);</string>
     </property>
     <property name="title">
      <string>下注区域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Zhuang">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 180, 255);</string>
        </property>
        <property name="text">
         <string>庄</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_Xian">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 180, 255);</string>
        </property>
        <property name="text">
         <string>闲</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ZTW">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 180, 255);</string>
        </property>
        <property name="text">
         <string>庄天王</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_XTW">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 180, 255);</string>
        </property>
        <property name="text">
         <string>闲天王</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_ZDZ">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 180, 255);</string>
        </property>
        <property name="text">
         <string>庄对子</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_XDZ">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 180, 255);</string>
        </property>
        <property name="text">
         <string>闲对子</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_Ping">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 180, 255);</string>
          </property>
          <property name="text">
           <string>平</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_TDP">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 180, 255);</string>
          </property>
          <property name="text">
           <string>同点平</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QListView" name="listView">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="horizontalFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_State">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="Poker" name="poker_1" native="true">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/poker_b/b_bg.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Poker" name="poker_2" native="true">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/poker_b/b_bg.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Poker" name="poker_3" native="true">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/poker_b/b_bg.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_Number">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../imgage.qrc">:/img/number/0.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_State">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>状态:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Poker" name="poker_4" native="true">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/poker_b/b_bg.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Poker" name="poker_5" native="true">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/poker_b/b_bg.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Poker" name="poker_6" native="true">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>80</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/poker_b/b_bg.png);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_head">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>119</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../imgage.qrc">:/img/wang.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_gold">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_Clip1">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>119</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 255);</string>
       </property>
       <property name="text">
        <string>筹码2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Clip2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>119</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>筹码3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Clip3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>119</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 110, 53);</string>
       </property>
       <property name="text">
        <string>筹码4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Clip4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>119</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 180, 157);</string>
       </property>
       <property name="text">
        <string>筹码5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Clip5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>119</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 170, 127);</string>
       </property>
       <property name="text">
        <string>筹码6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Poker</class>
   <extends>QWidget</extends>
   <header>base/UI/poker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../imgage.qrc"/>
 </resources>
 <connections/>
</ui>
